pkg/controllers/netpol/policy.go

```diff
@@ -262,55 +241,44 @@ func (npc *NetworkPolicyController) processIngressRules(policy networkPolicyInfo
 		if ingressRule.matchAllSource && ingressRule.matchAllPorts {
 			comment := "rule to ACCEPT traffic from all sources to dest pods selected by policy name: " +
 				policy.name + " namespace " + policy.namespace
-			if err := npc.appendRuleToPolicyChain(iptablesCmdHandler, policyChainName, comment, "", targetDestPodIPSetName, "", ""); err != nil {
+			if err := npc.appendRuleToPolicyChain(policyChainName, comment, "", targetDestPodIPSetName, "", ""); err != nil {
 				return err
 			}
 		}
 
 		if len(ingressRule.srcIPBlocks) != 0 {
 			srcIPBlockIPSetName := policyIndexedSourceIPBlockIPSetName(policy.namespace, policy.name, i)
-			srcIPBlockIPSet, err := npc.ipSetHandler.Create(srcIPBlockIPSetName, utils.TypeHashNet, utils.OptionTimeout, "0")
```

Comment:
We lost this in the conversion and then defaulted to TypeHashIP on all of our sets in the RefreshSet function: https://github.com/cloudnativelabs/kube-router/pull/1039/files#diff-7e4d6faaa20a55d353749c53d06bb4c9697a4498614bcd00997391b1a30af9f3R229

This causes all subnets to expand when sent to ipset which, for large subnets (subnets with addresses > 65,536) forces an error and stops the NetworkPolicyController from syncing.

Reply:
Fixed this with the commit I just pushed